### PR TITLE
Feature/frontend/profile page

### PR DESCRIPTION
**Pull request by tgnemecek**
In this PR I:
- Coded the FE layout for the dashboard/profile-page using placeholders for the data
- Added jsonconfig file to /client to allow absolute paths
Note: all previous imports still need to be updated

### CODE REVIEW

**Review by Hakeemmidan — APPROVED** [review]
Nice. For the Friend lists, you want to use the FriendList component that I created in [this PR](https://github.com/hatchways/team-eagle/pull/54). I would just pass down a bool prop to it that asks  it to not display the follow/unfollow button.

File: client/src/App.js [hunk 1]
```diff
@@ -2,11 +2,12 @@ import React from 'react';
...
 import { BrowserRouter, Route, Switch, Redirect } from 'react-router-dom';
 
-import { theme } from './themes/theme';
-import { UserContext } from './components/UserContext';
+import { theme } from 'themes/theme';
+import { UserContext } from 'components/UserContext';
 
-import NavBar from './components/NavBar';
-import LandingPage from './pages/Landing/Landing';
+import NavBar from 'components/NavBar';
```
Comment (Hakeemmidan) [inline client/src/App.js:8]:
```suggestion
import NavBar from './components/NavBar';
```

Comment (tgnemecek) [reply client/src/App.js:8]: Since my last merge we can use absolute imports

Comment (tgnemecek) [reply client/src/App.js:8]: Actually since THIS merge

**Pull request merged**